### PR TITLE
Make collon not managed by lsp

### LINKED ISSUE #1550: LSP-based highlighting of keyword namespace's `:` sign isn't consistent with how CIDER formats it
When I highlight my keyword using Emacs (CIDER and no LSP) then `:` sign is different from what the namespace color is.
When I do it on clojure-lsp-based highlighting mode then color of `:` is the same as namespace's color of that keyword.

CIDER on my Spacemacs:
image

With current LSP implementation (my nvim that uses clojure-lsp server):
image

**Describe the solution you'd like**
I think that `:` should be excluded from highlighting so that the editor's highlighting plugin would highlight it instead.

**Describe alternatives you've considered**
I used the https://github.com/clojure-vim/clojure.vim/ where I implemented the same `:` highlighting and it worked. But as nvim updated to 0.9.0 they now force the LSP highlighting and I don't yet know how to opt-out.

**Additional context**
```
> clojure-lsp --version
clojure-lsp 2023.02.27-13.12.12
clj-kondo 2023.02.17
```
```
> nvim --version
NVIM v0.9.0
```

### PR DESCRIPTION
**Pull request by victorpigmeo**
- [x] I created an issue to discuss the problem I am trying to solve or an open issue already exists.
- [x] I added a new entry to [CHANGELOG.md](https://github.com/clojure-lsp/clojure-lsp/blob/master/CHANGELOG.md)

Fixes #1550

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 - General
   - Add rewrite-clj node to cursor-info.
+  - Fixing semantic-tokens, collons not managed by lsp anymore.
```
Comment (ericdallo) [inline CHANGELOG.md:7]: Could you add the issue that fixes?
```suggestion
  - Fixing semantic-tokens, collons not managed by lsp anymore. #1234
```

File: lib/src/clojure_lsp/feature/semantic_tokens.clj [hunk 2]
```diff
@@ -150,22 +151,32 @@
...
 
 (defn ^:private elements->absolute-tokens
   [elements]
   (->> elements
        (sort-by (juxt :name-row :name-col))
        (keep
          (fn [{:keys [bucket] :as element}]
+           (when (= (:name-row element) 176)
+             (logger/info element))
+
```
Comment (ericdallo) [inline lib/src/clojure_lsp/feature/semantic_tokens.clj:179]: ?
```suggestion
```

Comment (victorpigmeo) [reply lib/src/clojure_lsp/feature/semantic_tokens.clj:179]: I removed this statement but I didn't push it 😅

**Review by ericdallo — APPROVED** [review]

**Comment by ericdallo** [comment]
Thank you!

**Pull request merged**